src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs

```diff
@@ -428,6 +440,11 @@ private async Task SetRunFromPackageAppSetting(Site functionApp, string runFromP
                 }
             }
 
+            if (functionApp.IsDynamic && functionApp.IsLinux && !functionApp.AzureAppSettings.ContainsKey("WEBSITE_MOUNT_ENABLED"))
+            {
+                functionApp.AzureAppSettings["WEBSITE_MOUNT_ENABLED"] = "1";
```

Comment:
Please correct me if I am wrong, but this will be set even when users are not in `--build-native-deps`, aka when we don't do SquashFS. Just making sure that's what you intended.

Reply:
Correct. Any new publish from the core-tools to consumption linux (zip or squashfs) will set this to `"1"`. Causing zip files to get mounted using `fuse-zip` and squashfs ones using `squashfuse`.

Reply:
but it'll leave it alone if the user flips this to `"0"` if they wanna disable the mounting behavior.